server/package.json

```diff
@@ -10,6 +10,7 @@
     "postinstall": "node ./cli/install_service_dependencies.js",
     "pretest": "npm run eslint",
     "test": "cross-env NODE_ENV=test ./node_modules/mocha/bin/mocha --recursive ./test/bootstrap.test.js \"./test/**/*.test.js\" --exit",
+    "test-system": "cross-env NODE_ENV=test ./node_modules/mocha/bin/mocha --recursive ./test/bootstrap.test.js \"./test/lib/system/*.test.js\" --exit",
```

Comment:
No need for that kind of thing, you can put `describe.only` or `it.only` during testing to only run one test.

Reply:
Yeah bad file commit, I was bored to run all tests to seen mine failed, thanks for the tips